packages/frint/src/App.ts

```diff
@@ -0,0 +1,410 @@
+import { Container, createContainer, Provider, resolveContainer } from 'frint-di';
+import { find, findIndex, get, omit } from 'lodash';
```

Comment:
we tend to avoid importing directly from the library to help reduce the bundle size of our custom applications.

this code below will import the whole lodash lib:

```js
import { get } from 'lodash';
```

this code below will import only the `get` function alone:

```js
import get from 'lodash/get';
```

with tree-shaking, we could eliminate this problem. but when the custom application is composed of various bundles coming in from multiple places (repos, apps), it makes things difficult with tree-shaking. since the bundler cannot see everything from a single entry point.